go.mod

```diff
@@ -4,6 +4,7 @@ go 1.17
 
 require (
 	github.com/google/go-cmp v0.5.6
+	github.com/hashicorp/go-plugin v1.4.3
```

Comment:
Note: This is temporary until terraform-plugin-go is updated: https://github.com/hashicorp/terraform-plugin-go/pull/123